tensorflow/compiler/xla/service/hlo_tfgraph_builder_test.cc

```diff
@@ -178,6 +178,23 @@ TEST_F(HloTfGraphBuilderTest, EmbeddedComputationsDiamond) {
   EXPECT_GT(generator_.GetGraphDef().node_size(), 0);
 }
 
+TEST_F(HloTfGraphBuilderTest, TokenHasNoLayout) {
+  auto builder = HloComputation::Builder("Token");
+  auto token = builder.AddInstruction(HloInstruction::CreateToken());
+  OpMetadata metadata;
+  metadata.set_op_name("x");
+  metadata.set_op_type("y");
+  token->set_metadata(metadata);
+  TF_CHECK_OK(generator_.AddComputation(*builder.Build()));
```

Comment:
TF_ASSERT_OK would be preferable.  CHECK will crash the whole test suite if it fails, including any other tests that haven't run yet.